components/brave_wallet/browser/asset_ratio_service.cc

```diff
@@ -94,6 +95,25 @@ std::vector<std::string> VectorToLowerCase(const std::vector<std::string>& v) {
   return v_lower;
 }
 
+constexpr char kEthereum[] = "ethereum";
+constexpr char kSolana[] = "solana";
+constexpr char kPolygon[] = "polygon";
+constexpr char kBitcoin[] = "bitcoin";
+
```

Comment:
These are file scope constants now and but actually only specific to ChainIdToStripeChainId function. Let's move them into that function, or even just inline into chain_id_lookup